src/tactic/lint/frontend.lean

```diff
@@ -151,7 +151,8 @@ let formatted_results := results.map $ λ ⟨linter_name, linter, results⟩,
 let s := format.intercalate "\n" (formatted_results.filter (λ f, ¬ f.is_nil)),
 let s := if verbose = lint_verbosity.low then s else
   format!("/- Checking {non_auto_decls.length} declarations (plus " ++
-  "{decls.length - non_auto_decls.length} automatically generated ones) {where_desc} -/\n\n") ++ s,
+  "{decls.length - non_auto_decls.length} automatically generated ones) {where_desc} " ++
+  "with {num_linters} linters -/\n\n") ++ s,
```

Comment:
I think probably listing all the linter names would be more useful here

Reply:
> I think listing all the linter names might be more useful here; unless the less verbose modes already handle that.

I think that will be too distracting/verbose and not too useful. 
All linters that complain are already displayed, and in the *more* verbose version, all linters (including the ones that raise no errors) display a message.
I thought this might be a nice tidbit to display.

> Is `num_linters` the same as `results.length`?

No, `results` only contains the linters that report an error.